blueflood-core/src/main/java/com/rackspacecloud/blueflood/inputs/processors/BatchWriter.java

```diff
@@ -109,6 +109,14 @@ else if (m instanceof PreaggregatedMetric)
                     } catch (Exception ex) {
                         getLogger().error(ex.getMessage(), ex);
                         getLogger().warn("Did not persist all metrics successfully for batch " + batchId);
+                        if ( getLogger().isDebugEnabled() ) {
+                            List<String> failedLocators = new ArrayList<String>() {{
+                                for (IMetric metric : batch) {
+                                    add(metric.getLocator().toString());
+                                }
+                            }};
+                            getLogger().debug("Failed batch contains: " + Arrays.toString(failedLocators.toArray()));
```

Comment:
@VinnyQ @ChandraAddala this is for you. Should I keep debug() or log as error() ?